modules/javascript/pages/ionic.adoc

```diff
@@ -10,28 +10,43 @@ include::partial$_set_page_context_for_javascript.adoc[]
 :param-related: xref:cordova.adoc[] | xref:react.adoc[]
 include::{root-partials}_show_page_header_block.adoc[]
 
+.Support Changes
+[IMPORTANT]
+
+--
+* The Ionic bridging-layer plugin is now open source and community supported.
+* The Couchbase Lite Native API functionality, which contains storage and sync functionality, shall be fully supported as normal for Enterprise customers. 
```

Comment:
No need to say Native as it might not be, e.g. cross-platform: The Couchbase Lite API, which contains core storage and sync functionality is fully supported for Enterprise customers.

Reply:
Done!